pkg/search/searcher_test.go

```diff
@@ -87,20 +103,64 @@ func TestSearcherCode(t *testing.T) {
 				firstRes := httpmock.JSONResponse(CodeResult{
 					IncompleteResults: false,
 					Items:             []Code{{Name: "file.go"}},
-					Total:             2,
+					Total:             1,
 				},
 				)
-				firstRes = httpmock.WithHeader(firstRes, "Link", `<https://api.github.com/search/code?page=2&per_page=100&q=org%3Agithub>; rel="next"`)
+				firstRes = httpmock.WithHeader(firstRes, "Link", `<https://api.github.com/search/code?page=2&per_page=30&q=org%3Agithub>; rel="next"`)
 				secondReq := httpmock.QueryMatcher("GET", "search/code", url.Values{
 					"page":     []string{"2"},
-					"per_page": []string{"29"},
+					"per_page": []string{"30"},
 					"q":        []string{"keyword language:go"},
 				},
 				)
 				secondRes := httpmock.JSONResponse(CodeResult{
 					IncompleteResults: false,
 					Items:             []Code{{Name: "file2.go"}},
-					Total:             2,
+					Total:             1,
+				},
+				)
+				reg.Register(firstReq, firstRes)
+				reg.Register(secondReq, secondRes)
+			},
+		},
```

Comment:
There are only 4 small changes in this tests.
`per_page` is 30, both in the header and the second request.

The total for the first and second responses have gone from 2 to 1.
This link to a previous review comment. I assume this total is the number of items in the response.
In that case, it has always been 1.